table/server/master/src/main/java/alluxio/master/table/Database.java

```diff
@@ -58,20 +60,23 @@
   private final String mName;
   private final Map<String, Table> mTables;
   private final UnderDatabase mUdb;
-  private final Map<String, String> mConfig;
+  private final CatalogConfiguration mConfig;
+  private final Set<String> mIgnoreTables;
   private final long mUdbSyncTimeoutMs =
       ServerConfiguration.getMs(PropertyKey.TABLE_CATALOG_UDB_SYNC_TIMEOUT);
 
   private DatabaseInfo mDatabaseInfo;
 
   private Database(CatalogContext context, String type, String name, UnderDatabase udb,
-      Map<String, String> configMap) {
+      CatalogConfiguration config) {
     mContext = context;
     mType = type;
     mName = name;
     mTables = new ConcurrentHashMap<>();
     mUdb = udb;
-    mConfig = configMap;
+    mConfig = config;
+    mIgnoreTables = Sets.newHashSet(Splitter.on(",").trimResults().omitEmptyStrings()
+        .split(mConfig.get(CatalogProperty.DB_IGNORE_TABLES)));
```

Comment:
This looks like the same implementation from `InstancedConfiguration`, but maybe we can extract this into a function in `FormatUtils` or `CommonUtils` so we don't have the code in both places?

Reply:
I extract that part.